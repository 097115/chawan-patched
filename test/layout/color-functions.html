<!DOCTYPE html>
<div style="color: rgba(100, 200, 30, 1)">
test
</div>
