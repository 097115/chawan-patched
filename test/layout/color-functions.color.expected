[38;2;100;200;30mtest[39m
